build.gradle

```diff
@@ -31,7 +31,6 @@ spotless {
 subprojects {
 	task allDeps(type: DependencyReportTask) {}
 	configurations.all {
-		exclude group: 'software.amazon.awssdk', module: 'netty-nio-client'
```

Comment:
This was originally excluded as a suggested best practice for Lambdas to not include httpclients (or any package really) that you're not using. It's brought in by the sdk automatically. We're using it now.